### PR TITLE
Adds dropdown menu to nav

### LINKED ISSUE #1861: [User Profile] Update top nav to link to profile
screen shot 2014-04-18 at 12 31 55 pm

@julielorch heads up this is a visualization for what we mentioned in the kick off meeting.

### LINKED ISSUE #3260: Remove "User Profile Link" theme setting / conditionals
Cleanup task for once we're ready to go live with the Profile. We don't need the theme setting or the conditional logic in the template anymore.

### PR DESCRIPTION
**Pull request by itsjoekent**
Fixes #1861 and Fixes #3260 (both are the same HTML/PHP so I killed two birds with one stone). If this is an issue ill update it.

CSS added to neue in https://github.com/DoSomething/neue/pull/443

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/templates/system/partials/navigation.tpl.php [hunk 1]
```diff
@@ -28,20 +28,21 @@
 
     <ul class="navigation__secondary">
       <li>
-        <?php print $search_box; ?>
+	       <?php print $search_box; ?>
```
Comment (aaronschachter) [inline lib/themes/dosomething/paraneue_dosomething/templates/system/partials/navigation.tpl.php]: Minor.. but adds weird spacing.

File: lib/themes/dosomething/paraneue_dosomething/templates/system/partials/navigation.tpl.php [hunk 2]
```diff
@@ -28,20 +28,21 @@
...
       </li>
       <?php if(!$logged_in): ?>
-      <?php // Will change 'Sign Up' to 'Create Account' once we refactor nav and can fit longer text! ?>
-      <?php if( theme_get_setting('show_profile_link') ): ?>
-      <li class="account"><a id="link--register" class="secondary-nav-item" href="<?php print $front_page; ?>user/register" data-modal-href="#modal--register"><?php print t('Register'); ?></a></li>
+        <?php if( theme_get_setting('show_profile_link') ): ?>
```
Comment (aaronschachter) [inline lib/themes/dosomething/paraneue_dosomething/templates/system/partials/navigation.tpl.php:34]: Is there an error in logic here? This looks like: If we are showing the profile link, display the login button.

Comment (itsjoekent) [inline lib/themes/dosomething/paraneue_dosomething/templates/system/partials/navigation.tpl.php:34]: I'm going to remove that conditional, Im confused how it managed to get back in there

**Comment by itsjoekent** [comment]
After I fix the logic error this is good to go (neue was released)

**Comment by aaronschachter** [comment]
@deadlybutter It may make sense to keep the conditional on the theme setting now for now. Once we've released to staging and get the green light from Team Product, remove the conditional in a separate PR.

**Comment by itsjoekent** [comment]
Yeah thats what I was gonna do for now, have a diff. PR for removing it when its time

**Comment by itsjoekent** [comment]
@aaronschachter all good :zap:

**Comment by aaronschachter** [comment]
As long as tested with both on/off and looks good, ship

**Pull request merged**